private-media-framework/draft-ietf-perc-private-media-framework.md

```diff
@@ -492,6 +492,14 @@ Key Distributor acts as the DTLS server.  The Key Encryption Key (KEK)
 association to endpoints via procedures defined in PERC EKT
 [@I-D.ietf-perc-srtp-ekt-diet] via the EKTKey message.
 
+In order to enable the Key Distributor to provide the KEK, the Key
```

Comment:
I'm a little confused.  By asking the KD to take the "active" role suggests it will act as the "client" and the endpoint as the "server".  Remember, the KD might not even the picture up to this point.  The KD cannot not send out the "ClientHello", nor should it be expected to establish the DTLS association between itself and the endpoint. The endpoint needs to initiate the association.  The DTLS packet flow will go from the endpoint to the MD, then over a tunnel to the KD.  In signaling, the endpoint should use "setup:actpass" and the MD (or other signaling function preparing an answer to the offer) should return "setup:passive".   The endpoint should send out the ClientHello and the KD should send the ServerHello.  And, it will send the EKTKey message at some point.  Using "passive" is a little slower, but the KD actually has no signaling path to the endpoint.

Reply:
Yes you are describing the correct scenario with KD. But I think the concern was that someone might want to use EKT in other scenarios as well. And if the standard then demands that always the DTLS server has to send the KEK that might cause problems.
Or maybe someone comes up with a design which actually allows the KD through already established tunnel to the MD, and with no NAT in between MD and endpoint, which allows the KD to act as the DTLS client.

Reply:
But this is the PERC framework.  Use of EKT in other scenarios should be defined elsewhere. Certainly, unless I'm confused, I don't think PERC would work as this is proposing.